src/graph/graphcontext.h

```diff
@@ -0,0 +1,78 @@
+#ifndef GRAPHCONTEXT_H
+#define GRAPHCONTEXT_H
+
+#include <assert.h>
+#include "../redismodule.h"
+#include "../index/index.h"
+#include "../stores/store.h"
+#include "graph.h"
+
+#define DEFAULT_INDEX_CAP 4
+
+typedef struct {
+  char *graph_name;                // String associated with graph
+  Graph *g;                        // Container for all matrices and entity properties
+
+  unsigned int relation_cap;       // Capacity of relation LabelStore array
+  unsigned int relation_count;     // Number of relation tyes
+  unsigned int label_cap;          // Capacity of node LabelStore array
+  unsigned int label_count;        // Number of label matrices.
+
+  LabelStore *relation_allstore;   // Schema for all relation types
```

Comment:
Can we rename relation to edge?

Reply:
You don't think that would be a little confusing? Our nomenclature is already a little dodgy around this area. Cypher's official position is that nodes have labels and relations have relation types.

'edge' makes me think of unique entities more than schemas (and Pieter's of the opinion that we should use either node-relation or vector-edge).